scripts/readstats.py

```diff
@@ -0,0 +1,93 @@
+#! /usr/bin/env python2
+#
+# This file is part of khmer, http://github.com/ged-lab/khmer/, and is
+# Copyright (C) Michigan State University, 2009-2013. It is licensed under
+# the three-clause BSD license; see doc/LICENSE.txt.
+# Contact: khmer-project@idyll.org
+#
+"""
+Display summary statistics for one or more FASTA/FASTQ files.
+
+% scripts/readstats.py [ -o output.txt ] <file1> <file2>
+
+Use '-h' for parameter help.
+"""
+
+import sys
+import screed
+import argparse
+import textwrap
+
+
+def main():
+    "Main function - run when executed as a script."
+    
+    descr = "Display summary statistics for one or more FASTA/FASTQ files."
+    epilog = ("""
+    Report number of bases, number of sequences, and average sequence length
+    for one or more FASTA/FASTQ files; and report aggregate statistics at end.
+
+    With :option:`-o`/:options:`--output`, the output will be saved to the
+    specified file.
+
+    Example::
+
+        readstats.py tests/test-data/test-abund-read-2.fa
+    """)
+
+    parser = argparse.ArgumentParser(description=descr,
+                                     epilog=textwrap.dedent(epilog))
+    parser.add_argument('filenames', nargs='+')
+    parser.add_argument('-o', '--output', dest='outfp',
+                        help="output file for statistics; defaults to stdout.",
+                        type=argparse.FileType('w'), default=None)
+    args = parser.parse_args()
```

Comment:
You have to expose your Argparse object via a method `get_parser()`. This is what is failing your build